src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,34 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * The {@code record} keyword.  This element appears
+     * as part of a record declaration.
+     **/
+    public static final int LITERAL_RECORD =
+            GeneratedJavaTokenTypes.LITERAL_record;
+
+    /**
+     * A record declaration.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * public record myRecord (String myString, String myOtherString) {}
```

Comment:
I do not see `myString` in AST example.
Please modify code example to be minimalistic, AST should be same by structure as our CLI is printing it.

Reply:
Fixed, I also updated the format of the AST to match the format we discussed in other PR.